tidy3d/web/__init__.py

```diff
@@ -11,16 +13,24 @@
     load,
     estimate_cost,
     abort,
+    get_tasks,
+    delete_old,
+    download_log,
+    download_json,
+    load_simulation,
+    real_cost,
+    test,
 )
-from .webapi import get_tasks, delete_old, download_log, download_json, load_simulation, real_cost
-from .container import Job, Batch, BatchData
 from .cli import tidy3d_cli
-from .cli.app import configure_fn as configure
-from .asynchronous import run_async
-from .webapi import test
+from .api.asynchronous import run_async
+from .core import core_config
+from ..log import log, get_logging_console
+from ..version import __version__
 
 migrate()
 
+core_config.set_config(log, get_logging_console(), __version__)
```

Comment:
@dbochkov-flexcompute does passing the `Console()` instance seem to work ok in the notebooks?

Reply:
this is not ideal, but works ok in notebooks and I don't think we will have issues with this